modules/cluster-logging-updating-logging-to-46.adoc

```diff
@@ -0,0 +1,321 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging_{context}"]
+= Updating cluster logging
+
+[role="_abstract"]
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+When you update:
+
+* You must update the Elasticsearch Operator before updating the Cluster Logging Operator.
+* You must update both the Elasticsearch Operator and the Cluster Logging Operator.
++
+Kibana is unusable when the Elasticsearch Operator has been updated but the Cluster Logging Operator has not been updated.
++
+If you update the Cluster Logging Operator before the Elasticsearch Operator, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, delete the Cluster Logging Operator pod. When the Cluster Logging Operator pod redeploys, the Kibana CR is created.
+
+[IMPORTANT]
+====
+If your cluster logging version is prior to 4.5, you must upgrade cluster logging to 4.5 before updating to 4.6.
+====
+
+.Prerequisites
+
+* Update the {product-title} cluster from 4.4 to 4.6.
+
+* Make sure the cluster logging status is healthy:
++
+** All pods are `ready`.
+** The Elasticsearch cluster is healthy.
+
+* Back up your Elasticsearch and Kibana data.
+
+.Procedure
+
+. Update the Elasticsearch Operator:
+
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-operators-redhat` project.
+
+.. If the Elasticsearch Operator is installed on the `openshift-logging` names space, uninstall the Elasticsearch Operator.
+
+[NOTE]
+====
+This uninstall action does _not_ delete the existing data, volumes, and so on.
+====
+
+.. Click *Operators* -> *OperatorHub*. Search for and select OpenShift Elasticsearch Operator.
+
+ .. Click *Install*
+ +
+ The Install operator page opens.
```

Comment:
Is it labeled as Install operator? If so, we should put it in bold as:

     *Install operator* page

Reply:
Done.